### PR TITLE
Updated pymc.Normal docstring

### PR DESCRIPTION
**Pull request by sagartomar**
**Thank your for opening a PR!**

Depending on what your PR does, here are a few things you might want to address in the description:
+ [ ] what are the (breaking) changes that this PR makes?
+ [ ] important background, or details about the implementation
+ [ ] are the changes—especially new features—covered by tests and docstrings?
+ [x] [linting/style checks have been run](https://docs.pymc.io/en/latest/contributing/python_style.html)
+ [ ] [consider adding/updating relevant example notebooks](https://github.com/pymc-devs/pymc-examples)
+ [ ] right before it's ready to merge, mention the PR in the RELEASE-NOTES.md

For issue #5459 

#DataUmbrellaPyMCSprint

### CODE REVIEW

File: pymc/distributions/continuous.py [hunk 1]
```diff
@@ -524,11 +525,11 @@ class Normal(Continuous):
 
     Parameters
     ----------
-    mu: float
-        Mean.
-    sigma: float
+    mu : float
+        Mean. Defaults to 0.
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]: the default should go in the above line here as it's of the documented type and fixed. reference https://pymc-data-umbrella.xyz/en/latest/sprint/docstring_tutorial.html#parameters

File: pymc/distributions/continuous.py [hunk 2]
```diff
@@ -573,9 +574,13 @@ def logcdf(value, mu, sigma):
...
         Parameters
         ----------
-        value: numeric or np.ndarray or `TensorVariable`
+        value : numeric or ndarray or TensorVariable
             Value(s) for which log CDF is calculated. If the log CDF for multiple
             values are desired the values must be provided in a numpy array or `TensorVariable`.
+        mu : float
+            Mean. Defaults to 0.
+        sigma : float
+            Standard deviation (sigma > 0).
```
Comment (Sayam753) [inline pymc/distributions/continuous.py]: I think `mu` and `sigma` parameters can be numpy array/tensor variables as well.

File: pymc/distributions/continuous.py [hunk 3]
```diff
@@ -524,11 +525,11 @@ class Normal(Continuous):
 
     Parameters
     ----------
-    mu: float
-        Mean.
-    sigma: float
+    mu : float
+        Mean. Defaults to 0.
+    sigma : float
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]:
```suggestion
    sigma : float, optional
```

here the default is not immediate as it depends on the value of tau too, so it should use optional only with the default on the description. In this case the description should mention that when no value is passed for tau nor sigma, sigma defaults to one

File: pymc/distributions/continuous.py [hunk 4]
```diff
@@ -524,11 +525,11 @@ class Normal(Continuous):
...
         Standard deviation (sigma > 0) (only required if tau is not specified).
-    tau: float
+    tau : float
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]: same, optional here

File: pymc/distributions/continuous.py [hunk 5]
```diff
@@ -524,11 +525,11 @@ class Normal(Continuous):
 
     Parameters
     ----------
-    mu: float
-        Mean.
-    sigma: float
+    mu : float
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]: I think the type should be `numeric, array_like or TensorVariable` cc @Sayam753 @twiecki @michaelosthege

File: pymc/distributions/continuous.py (hunk 2 above)
Comment (ricardoV94) [reply pymc/distributions/continuous.py]: I think we only mention the base case

Comment (ricardoV94) [reply pymc/distributions/continuous.py]: numpy uses float or array_like of floats: https://numpy.org/doc/stable/reference/random/generated/numpy.random.normal.html

Comment (OriolAbril) [reply pymc/distributions/continuous.py]: I have a similar comment below, missed this one. I'd propose something like `float, array_like or TensorVariable` not sure if we need to add `of float` to both array and tensor types :thinking:

Comment (sagartomar) [reply pymc/distributions/continuous.py]: `float, array_like or TensorVariable` looks good to me. Should I update the pull request with this?

Comment (OriolAbril) [reply pymc/distributions/continuous.py]: Use `tensor_like of float`. We will update the glossary to have a definition for the type like numpy does with array_like.

File: pymc/distributions/continuous.py [hunk 6]
```diff
@@ -524,12 +525,14 @@ class Normal(Continuous):
...
+    mu : tensor_like of float, default 0
         Mean.
-    sigma: float
+    sigma : tensor_like of float, optional
         Standard deviation (sigma > 0) (only required if tau is not specified).
-    tau: float
+        Defaults to 1 if neither sigma nor tau is specified.
+    tau : tensor_like of float, optional
         Precision (tau > 0) (only required if sigma is not specified).
+        Defaults to 1 if neither sigma nor tau is specified.
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]:
```suggestion
```

only one of tau or sigma are used, I think it's better to leave the explanation as no value for tau or sigma -> sigma=1 (tau=1 is equivalent to sigma=1 but that might still be confusing for many pymc users)

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**